app/services/evacancy/ev_connect_service.rb

```diff
@@ -0,0 +1,64 @@
+module Evacancy
+  class EvConnectService
+    class << self
+      def check_ports
+        token = check_access_token
+
+        previously_open = 0
+        currently_open = 0
+
+        EvConnectStationPort.all.each do |station|
+          new_status = get_status(token, station.qr_code)
+          previously_open += 1 if station.port_status == 'AVAILABLE'
```

Comment:
Not sure if you envision these status strings being referenced anywhere else, but if so it might be nice to create some constants for them.